demos/index.html

```diff
@@ -62,7 +72,28 @@
                                 width: 400,
                                 height: 400
                             }
-                        }
```

Comment:
By "moving into components", do you mean we should assess whether or not some of these options should become the default options of the actual components involved in the chartAuthoring component, rather than being configured this way for the demo?